tests/e2e/e2e.sh

```diff
@@ -17,83 +17,54 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
+HUB="istio-testing"
+TAG="istio-testing"
+ROOT=$(cd ../../../; pwd)
 
-function cleanup_kind_cluster() {
-    kind export logs --name istio-testing "${ARTIFACTS}/kind"
-    kind delete cluster --name=istio-testing
-}
-
-function setup_kind_cluster() {
-  # Delete any previous e2e KinD cluster
-  echo "Deleting previous KinD cluster with name=istio-testing"
-  if ! (kind delete cluster --name=istio-testing) > /dev/null; then
-    echo "No existing kind cluster with name istio-testing. Continue..."
-  fi
-
-  trap cleanup_kind_cluster EXIT
-
-  # Create KinD cluster
-  if ! (kind create cluster --name=istio-testing --loglevel debug --retain --wait 30s); then
-    echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
-    exit 1
-  fi
-
-  KUBECONFIG="$(kind get kubeconfig-path --name="istio-testing")"
-  export KUBECONFIG
-
-cat <<EOF > metallb-config.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  namespace: metallb-system
-  name: config
-data:
-  config: |
-    address-pools:
-    - name: default
-      protocol: layer2
-      addresses:
-      - 172.17.255.1-172.17.255.250
-EOF
-
-  kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.8.1/manifests/metallb.yaml
-  kubectl apply -f ./metallb-config.yaml
-}
 
 function setup_docker() {
-  HUB=istio-testing TAG=latest make -f Makefile.core.mk controller docker
-  kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:istio-testing
+  HUB=gcr.io/istio-testing TAG=1.5-dev make controller docker
+  kind --loglevel debug --name istio-testing load docker-image gcr.io/istio-testing/operator:1.5-dev
+  kind --loglevel debug --name istio-testing load docker-image istio-testing/app:istio-testing
```

Comment:
where is app coming from?

Reply:
nevermind, I see we build it later, but probably don't need to

Reply:
we need this for simple testing, especially the testNgix part